cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/SportResults.java

```diff
@@ -37,11 +37,11 @@ public void configure() {
     createDataset("results", PartitionedFileSet.class, PartitionedFileSetProperties.builder()
       // properties for partitioning
       .setPartitioning(Partitioning.builder().addStringField("league").addIntField("season").build())
-        // properties for file set
+      // properties for file set
       .setInputFormat(TextInputFormat.class)
       .setOutputFormat(TextOutputFormat.class)
       .setInputProperty(TextOutputFormat.SEPERATOR, ",")
```

Comment:
Looks like from this (https://hadoop.apache.org/docs/r2.3.0/api/src-html/org/apache/hadoop/mapreduce/lib/output/TextOutputFormat.html#line.45), that this is the source of the mis-spelling.